corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -21,6 +21,9 @@
 
 
 def validate_phone_datetime(datetime_string, none_ok=False, form_id=None):
+    if isinstance(datetime_string, datetime.datetime):
```

Comment:
@czue : its definitely happening from [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/tasks.py#L298) since forms are fetched via FormAccessors resulting in already parsed `datetime.datetime` object and not a string.
I am not sure but i guess this was recently added as shared by Robin "I was testing to see the new multimedia export ....".

Returning the value if its already a datetime.datetime object seems ok even otherwise. Though the argument name `datetime_string` conflicts.
If we want to make this better we can either change [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/case/xform.py#L373) to validate only if its a string or if its not a datetime.datetime object already (preferred since None and empty string need are handled differently).

I think we are missing test cases for this hence it was not noticed earlier.